src/neptune_fetcher/alpha/filters.py

```diff
@@ -69,6 +122,51 @@ def __or__(self, other: "BaseAttributeFilter") -> "BaseAttributeFilter":
 
 @dataclass
 class Attribute:
+    """Helper for specifying an attribute and picking a metric aggregation function.
+
+    Use to build filters for fetching experiments or runs.
+
+    Args:
+        name (str): An attribute name to match exactly.
+        aggregation (Literal["last", "min", "max", "average", "variance"], optional):
+            Aggregation function to apply when specifying a metric of type FloatSeries.
+            Defaults to `"last"`, i.e. the last logged value.
+        type (Literal["float", "int", "string", "bool", "datetime", "float_series", "string_set"], optional):
+            Allowed attribute type. For a reference, see: https://docs-beta.neptune.ai/attribute_types
```

Comment:
It's not as much `Allowed attribute type` as just `Attribute type`. This arg enables users to explicitly state the type in case the type cannot be inferred automatically, or they want to write future-proof script (i.e. even if now the type can be inferred, in the future it might become ambiguous if some experiment/run adds an attribute of the same name but different type).

Reply:
I see. For clarity, we should spell it out:

```suggestion
            Attribute type. Specify it to resolve ambiguity, in case some of the project's runs contain attributes
            that have the same name but are of a different type.
            For a reference, see: https://docs-beta.neptune.ai/attribute_types
```